### ISSUE TITLE
Issue in swagger migration guide

### ISSUE THREAD

**Issue by delenius** [OpenAPITools/openapi-generator#150]
In [the swagger-codegen migration guide](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/migration-from-swagger-codegen.md), there is this confusing section. I'm actually not sure what it's supposed to say, or I would submit a PR:

```
Renamed Mustache Template Variables
The template variable {{datatype}} was renamed to {{dataType}} for consistency reason. Corresponding java code: CodegenProperty.datatype is renamed to CodegenProperty.datatype.
```

**Comment by wing328** [comment]
@delenius thanks for the feedback.

That section aims to keep our users up-to-date regarding some mustache variable renaming. If you're not using customized templates, you do not need to worry about that.

I'll add a line to explain that's for using customized templates only.

**Comment by delenius** [comment]
Yes but it says "replace X with X".

**Comment by delenius** [comment]
Oh, I see the lower vs uppercase "t" now, but only in the first case. It still says `CodegenProperty.datatype` twice.

**Comment by wing328** [comment]
Good catch. I'll fix that.

**Comment by wing328** [comment]
UPDATE: I've filed https://github.com/OpenAPITools/openapi-generator/pull/152 to fix it.

**Comment by wing328** [comment]
Closed via #152. Thanks again for the feedback.
